packages/theme/src/cli/commands/theme/dev.ts

```diff
@@ -64,13 +64,15 @@ export default class Dev extends ThemeCommand {
     }),
   }
 
+  static cli2Flags = ['host', 'live-reload', 'poll', 'theme-editor-sync', 'port', 'theme', 'only', 'ignore', 'stable']
```

Comment:
As I merged https://github.com/Shopify/cli/pull/751 _before_, the `--force` flag got introduced for `theme dev/pull/push/share`.

I therefore added the necessary changes and created a follow-up PR: https://github.com/Shopify/cli/pull/764

You can either cherry-pick my changes onto this PR's branch, or review https://github.com/Shopify/cli/pull/764

Thanks!

Reply:
Thanks for the note, @Poitrin! I finally got around to 🎩 and added a few `force`s, so maybe just rebase/merge against main and make sure I haven't missed anything.